### ISSUE TITLE
Simpler, more universal language

### ISSUE THREAD

**Issue by Mr0grog** [rosshinkley/nightmare-examples#4]
Mapbox recently released a really great tool they use for linting their documentation: https://www.mapbox.com/blog/retext-mapbox-standard/

Obviously there are a lot of specific-to-Mapbox bits in there, but it could still be used here to clean up language:

```
CHANGELOG.md: no issues found

README.md
  20:1-20:7    warning  Remove “Simply”              simply
  20:68-20:74  warning  Replace “all of” with “all”  all of

docs/beginner/co.md
  38:29-38:35  warning  Remove “easily”             easily

docs/beginner/promises.md
  26:93-26:99  warning  Replace “modify” with “change”  modify

docs/beginner/semantic-callbacks.md: no issues found
docs/beginner/vo.md: no issues found

docs/known-issues/uploads.md
  2:165-2:175  warning  Replace “similar to” with “like”              similar to
   5:1-5:10    warning  Replace “Currently” with “now”, or remove it  currently

docs/known-issues/then-catch.md
  2:207-2:222    warning  Replace “in an effort to” with “to”  in an effort to
  2:223-2:229    warning  Replace “ensure” with “make sure”    ensure
  21:71-21:78    warning  Replace “However” with “but”, “yet”  however
  24:113-24:119  warning  Replace “As yet” with “yet”          as yet

⚠ 10 warnings
```

**Comment by rosshinkley** [comment]
Ah, that's a good tool to know about.  I agree the language should be straightforward.  I'll fold these changes in.

**Comment by rosshinkley** [comment]
Fixed everything except for "as yet" in #12.
